src/gebt_poc/mesh.h

```diff
@@ -0,0 +1,100 @@
+#pragma once
+
+#include <Kokkos_Core.hpp>
+
+namespace openturbine::gebt_poc {
+class Mesh {
+public:
+    int GetNumberOfElements() const { return number_of_elements_; }
+
+    int GetNumberOfNodes() const { return number_of_nodes_; }
+
+    KOKKOS_FUNCTION
+    Kokkos::View<const int*> GetNodesForElement(int elementID) const {
```

Comment:
Let's rename `elementID` -> `element_id` or `elem_id` to be consistent with the style guide.